acvm/src/lib.rs

```diff
@@ -227,9 +227,7 @@ pub fn checksum_constraint_system(cs: &Circuit) -> u32 {
 // This is set to match the previous functionality that we had
 // Where we could deduce what opcodes were supported
 // by knowing the np complete language
-pub fn default_is_opcode_supported(
-    language: Language,
-) -> compiler::transformers::IsOpcodeSupported {
+pub fn default_is_opcode_supported(language: Language) -> impl Fn(&Opcode) -> bool {
```

Comment:
```suggestion
pub fn default_is_opcode_supported(language: Language) -> fn(&Opcode) -> bool {
```

We can make this type stricter as we're not going to be capturing the environment here.

Reply:
Made this change and tested that this is still accepted as a `impl Fn(&Opcode) -> bool` as a sanity check.